LICENSE

```diff
@@ -1,4 +1,4 @@
-The MIT License
+The MIT License (MIT)
```

Comment:
Good spot. Change the copyright below to 2013 - 2016 Peter Elger and other contributors